packages/mobile_app/lib/routes.dart

```diff
@@ -33,7 +33,7 @@ class InitRoute extends StatelessWidget {
 
               //
 
-              final usersRepo = context.watch<UsersRepositoryFlutter>();
+              final usersRepo = authBloc.auth.repository;
```

Comment:
Much, much faster then wait for context item. (0.03s)

Reply:
I dont really understand the difference, so thats fine :)